composer.json

```diff
@@ -20,13 +20,13 @@
     ],
     "require": {
         "php": "^8.1",
-        "laravel/framework": "^10.16|^11.0",
+        "laravel/framework": "^10.17|^11.0",
         "livewire/livewire": "^3.0"
     },
     "require-dev": {
         "laravel/folio": "^1.0",
         "mockery/mockery": "^1.6",
-        "orchestra/testbench": "^8.5",
+        "orchestra/testbench": "^8.6.2",
```

Comment:
`^8.6.2` have `^10.17` as minimum, it might be best to bump `laravel/framework` as well.